pkg/k8shandler/visualization.go

```diff
@@ -53,15 +53,15 @@ func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateVisualization(proxyCo
 func (clusterRequest *ClusterLoggingRequest) UpdateKibanaStatus() (err error) {
 	kibanaStatus, err := clusterRequest.getKibanaStatus()
 	if err != nil {
-		logrus.Errorf("Failed to get Kibana status for %q: %v", clusterRequest.Cluster.Name, err)
+		log.Error(err, "Failed to get Kibana status for", "clusterName", clusterRequest.Cluster.Name, err)
```

Comment:
this looks like `"clusterName"` maybe could be removed?

Reply:
I believe "clusterName" is accurate. You left a dangling `err` at the end though